src/nlp/dataset_dict.py

```diff
@@ -16,6 +17,71 @@ def _check_values_type(self):
                     "Values in `DatasetDict` should of type `Dataset` but got type '{}'".format(type(dataset))
                 )
 
+    @contextlib.contextmanager
+    def formated_as(
```

Comment:
You can add these methods to the docs in `docs/source/package_reference/main_classes.rst`

Reply:
😄